package.json

```diff
@@ -88,11 +88,11 @@
         "eslint-plugin-scanjs-rules": "^0.2.1",
         "eslint-plugin-security": "^1.4.0",
         "jest": "^26.5.3",
-        "prettier": "^2.1.2",
+        "prettier": "^2.6.2",
         "rollup": "^2.38.1",
         "rollup-plugin-terser": "^7.0.2",
         "tmp": "^0.2.1",
-        "typescript": "^4.1.3"
+        "typescript": "~4.4.4"
```

Comment:
We are going with the '~', because versions higher than `4.4.x` are not as compatible and would require us to do put some extra effort into migration?

Just wanted to make sure that I understand the reasoning behind that decision 😁

Reply:
Yes, this is the case

Reply:
Okay, thanks for the confirmation 💪